lightning/src/ln/channelmanager.rs

```diff
@@ -6489,6 +6489,9 @@ where
 											user_channel_id: chan.context.get_user_id(),
 											new_funding_txo: splice_negotiated.funding_txo,
 											channel_type: splice_negotiated.channel_type,
+											new_funding_redeem_script: chan
+												.funding
+												.get_funding_redeemscript(),
```

Comment:
This will still be the old funding until the splice is promoted. You'll need to communicate it through `SpliceFundingNegotiated`.